src/features/cartography/infrastructure/configuration/modules/cartography.feature.module.ts

```diff
@@ -24,6 +36,10 @@ import { GeocodeAddressUseCase } from '../../../use-cases/geocode-address/geocod
     },
     CnfsRest,
     CoordinatesRest,
+    {
+      provide: ClusterService,
+      useFactory: (): ClusterService => new ClusterService(ClusterService.initCluster())
```

Comment:
```suggestion
      useFactory: (): ClusterService => new ClusterService()
```
avec `ClusterService.initCluster()` appelé dans le constructeur